<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="choice_map">
     <item>
      <widget class="QRadioButton" name="l_map">
       <property name="text">
        <string>Схема</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="sat_map">
       <property name="text">
        <string>Спутник</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="hybrid_map">
       <property name="text">
        <string>Гибрид</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="picture">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>650</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
